index.ts

```diff
@@ -202,9 +202,10 @@ export const isOwnUserProfile = (): boolean => getCleanPathname() === getLoggedI
 // If there's a Report Abuse link, we're not part of the org
 export const isOwnOrganizationProfile = (): boolean => isOrganizationProfile() && !exists('[href*="contact/report-abuse?report="]');
 
-export const isProject = (url: URL | HTMLAnchorElement | Location = location): boolean => /^projects\/\d+/.test(getRepo(url)?.path);
+export const isProject = (url: URL | HTMLAnchorElement | Location = location): boolean => /^projects\/\d+/.test(getRepo(url)?.path ?? getOrg(url)?.path);
 TEST: addTests('isProject', [
 	'https://github.com/sindresorhus/refined-github/projects/3',
```

Comment:
```suggestion
	'https://github.com/orgs/eslint/projects/4',
```

Reply:
If you apply this change, most likely you'll have to update the URL to other parts as well